composer.json

```diff
@@ -8,35 +8,26 @@
             "email": "info@palantir.net"
         }
     ],
-    "minimum-stability": "dev",
-    "prefer-stable": true,
-    "config": {
-        "platform": {
-            "php": "7.2"
-        },
-        "preferred-install": {
-            "*": "dist"
-        },
-        "sort-packages": true
-    },
     "repositories": [
         {
             "type": "composer",
             "url": "https://packages.drupal.org/8"
         }
     ],
     "require": {
-        "composer/installers": "^1.0",
-        "drupal/core-composer-scaffold": "^8.8",
+        "composer/installers": "^1.2",
+        "drupal/coder": "^8.3.6",
```

Comment:
I'd prefer not to add drupal/coder back in here, and leave it as a dependency of the-build.

Reply:
oops. I only added coder because I saw it before.
By mistake I looked at an older version of `composer.json`...
https://github.com/palantirnet/drupal-skeleton/commit/93fced74ce60947c1e500ceceb7e3f77bea269f6

I agree it's a great idea to remove it here, and let `the-build` add it.